components/partials/ArticleFooter.vue

```diff
@@ -1,17 +1,194 @@
 <template>
-  <footer style="height: 200px; background-color: lightgray;">
-    Tags, about author and social buttons
+  <footer class="article-footer">
+    <ul class="article-footer__tags">
+      <li v-for="(tag, index) in tags" class="article-footer__single-tag" :key = "index">
+        <router-link to="" class="article-footer__single-tag-route"> {{ tag }} </router-link>
+      </li>
+    </ul>
+    <div class="article-footer__footer-box">
+      <img class="article-footer__author-avatar" :src="author.avatar.url"/>
```

Comment:
Avatar should link to the author's profile. Please wrap it in a `router-link` tag. Add also `:alt="author.fullname"` attribute.